app/views/assessor_interface/review_verifications/index.html.erb

```diff
@@ -7,17 +7,20 @@
   The following verification tasks have been flagged for review:
 </p>
 
-<%= govuk_table do |table|
-  if @professional_standing_request.present?
```

Comment:
Although we only support LoPS at the moment we'll need this condition in the future when we implement review of qualifications and references so I think it makes sense to keep it here.